Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m

```diff
@@ -101,6 +107,8 @@ - (UIColor *)highlightColor
 
 - (void)createContents
 {
+    OWSAssert(!self.boundsStrokeView);
```

Comment:
A comment could clarify why we're doing this: `// ensure called only once`

Reply:
Done.